pkg/gomod/parse_testcase.go

```diff
@@ -0,0 +1,388 @@
+package gomod
+
+import "github.com/aquasecurity/go-dep-parser/pkg/types"
+
+var (
+	// docker run --name gomod --rm -it golang:1.15 bash
+	// export USER=gomod
+	// mkdir repo
+	// cd repo
+	// go mod init github.com/org/repo
+	// go get golang.org/x/xerrors
+	// go list -m all | awk 'NR>1 {sub(/^v/, "", $2); sub("-.*", "", $2); printf("{\""$1"\", \""$2"\"},\n")}'
+	GoModNormal = []types.Library{
+		{"golang.org/x/xerrors", "0.0.0"},
+	}
+
+	// docker run --name gomod --rm -it golang:1.15 bash
+	// export USER=gomod
+	// mkdir repo
+	// cd repo
+	// go mod init github.com/org/repo
+	// go get golang.org/x/xerrors
+	// go get github.com/urfave/cli
+	// go get github.com/stretchr/testify
+	// go get github.com/BurntSushi/toml
+	// go list -m all | awk 'NR>1 {sub(/^v/, "", $2); sub("-.*", "", $2); printf("{\""$1"\", \""$2"\"},\n")}'
```

Comment:
1. using `go list -m` since it is a module-aware command and output is easier to parse than `cat go.sum`
2. ignoring first line since it is the name of the main module, which is ignored in `go.sum`

Reply:
Sounds great!